### ISSUE TITLE
Not running in evaluation mode during testing/generation intentional?

### ISSUE THREAD

**Issue by tonypeng** [DmitryUlyanov/texture_nets#35]
It seems to me that the network is not being run in evaluation mode (and therefore use the population mean/variance for the batch/instance normalization layers) when running feed forward/generating styled images. Is this intentional? I read your paper and wasn't sure if 

> Differently from batch normalization, furthermore, the instance normalization layer is applied at test time as well

meant to calculate the means/variances during test time as well/not use population statistics.

**Comment by DmitryUlyanov** [comment]
Yes, batch norm in training mode and batch_size = 1 is instance norm.
